test/unit/api/scripts/_languages.js

```diff
@@ -0,0 +1,27 @@
+define(function (require) {
+  var expect = require('intern/dojo/node!expect.js');
+
+  return function (bdd, request) {
+    bdd.describe('Languages API', function getLanguages() {
+
+      bdd.it('should return 200 with an array of languages', function () {
+        return request.get('/kibana/scripts/languages')
+        .expect(200)
+        .then(function (response) {
+          expect(response.body).to.be.an('array');
```

Comment:
I haven't looked much at this kind of test code before. Does it actually query the server connected to Elasticsearch and get an actual response, or are there mocks somewhere?